client/sites/staging-site/components/staging-site-sync-modal/index.tsx

```diff
@@ -398,7 +398,11 @@ export default function SyncModal( {
 						<Button variant="tertiary" onClick={ onClose }>
 							{ __( 'Cancel' ) }
 						</Button>
-						<Button variant="primary" onClick={ handleConfirm }>
+						<Button
+							variant="primary"
+							onClick={ handleConfirm }
+							disabled={ ! browserCheckList.totalItems }
```

Comment:
I think the `browserCheckList` might be undefined by default, until the checkboxes are interacted with. Even though the "Files and folders" checkbox is checked, the CTA remains disabled for me when first loading the modal.

https://github.com/user-attachments/assets/928c6e22-c3ff-4f78-a7ef-096cdbec020a

Roberto also addressed this in https://github.com/Automattic/wp-calypso/pull/104779, so this might not be an issue once that PR is merged.